packages/cadl-ranch-specs/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @azure-tools/cadl-ranch-specs
 
+## 0.34.10
+
+### Patch Changes
+
+- 1cdf0c8: updated dependencies @azure-tools/cadl-ranch-api@0.4.4
```

Comment:
we will have to release a new library for cadl-ranch-specs as the cadl-ranch is released.